### PR TITLE
Structure2

### PR DESCRIPTION
**Pull request by hedita**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,145 +16,127 @@
   <h1 class="page-header">Movies</h1>
   <ul class="movie-list">
     <li class="movie-item">
-      <img class="img" src="./assets/2.jpg">
+      <img class="img" src="./assets/img/img.jpg">
       <div class="movie-details">
```
Comment (takrami) [inline index.html]: they can be sections.

File: index2.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+<head>
+    <link rel="stylesheet" href="./assets/css/structure2.css" />
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Movie</title>
+</head>
+<body>
+  <h1 class="title">Movies</h1>
+  <div class="contant">
```
Comment (takrami) [inline index2.html]: can we remove this div?

File: index2.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    <div class="movie-section">
```
Comment (takrami) [inline index2.html]: how about this one?

File: index2.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        <img class="img" src="./assets/img/img.jpg">
+    <div class="details-container">
```
Comment (takrami) [inline index2.html]: can we remove this div?

File: index2.html [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+        <section class="movie-details">
+        <h2 class="movie-name">Name</h2>
+        <p class="rate">7/10</p>
+        </section>
+        <p class="summary">summary</p>
+    </div>
+    </div>
+  </div>
+</body>
+</html>
```
Comment (takrami) [inline index2.html]: please fix indentation for this file.

Comment (hedita) [reply index2.html]: Done.

File: index.html [hunk 6]
```diff
@@ -16,144 +16,184 @@
...
-    </li>
-    <li class="movie-item">
-      <img class="img" src="./assets/3.jpg">
-      <div class="movie-details">
-        <h2 class="movie-name">Movie</h2>
-        <p class="movie-release">release</p>
-      </div>
+      <a href="./movie.html">
+        <img class="img" src="./assets/img/img.jpg">
+        <div class="movie-details">
```
Comment (takrami) [inline index.html]: it can be changed to a section.

File: movie.html [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+  </head>
+  <body>
+    <a href="./index.html" class="title">Movies</a>
+    <figure class="details-container">
+      <img class="img" src="./assets/img/img.jpg">
+      <figcaption class="details">
+        <h2 class="movie-name">Name</h2>
+        <p class="rate">7/10</p>
+        <p class="summary">summary</p>
+      </figurcaption>
```
Comment (takrami) [inline movie.html]: please fix this.

File: assets/css/movie-structure.css [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+.movie-name {
+  font-size: x-large;
+  display: inline;
+}
+
+.rate {
+  color: var(--secondery-text-color);
+  font-size: bold;
```
Comment (takrami) [inline assets/css/movie-structure.css]: please use variables for font size.

File: index.html (hunk 6 above)
Comment (hedita) [reply index.html]: Done.

File: assets/css/movie-structure.css (hunk 8 above)
Comment (hedita) [reply assets/css/movie-structure.css]: Done.

File: movie.html (hunk 7 above)
Comment (hedita) [reply movie.html]: Fixed.

File: assets/css/general.css [hunk 9]
```diff
@@ -5,5 +5,5 @@
 }
 
 body {
```
Comment (takrami) [inline assets/css/general.css]: move this to the structure file. then you can remove this file.

Comment (hedita) [reply assets/css/general.css]: Done.

File: assets/css/main.css [hunk 10]
```diff
@@ -1,4 +1,5 @@
 @import "./general.css";
```
Comment (takrami) [inline assets/css/main.css]: remove this. you don't have this file anymore.

File: movie.html (hunk 7 above)
Comment (takrami) [reply movie.html]: this is the correct tag: figcaption

File: assets/css/movie-structure.css [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+  font-size: var(--secondery-font-size);
+  padding-top: var(--spacing-large);
+  padding-left: var(--spacing-small);
+}
+
+/* Mobile */
+@media(max-width: 600px) {
+ 
+  .details-container {
+    display: grid;
```
Comment (takrami) [inline assets/css/movie-structure.css:62]: please make the image and content center.

**Review by takrami — APPROVED** [review]
perfect!  :+1:  just we need to fix the fonts when the internet is better.

**Pull request merged**